Paintroid/src/main/java/org/catrobat/paintroid/tools/implementation/CursorTool.java

```diff
@@ -58,6 +61,7 @@ public CursorTool(Context context, ToolType toolType) {
 				.getResources().getColor(
 						R.color.cursor_tool_deactive_primary_color);
 		mSecondaryShapeColor = Color.LTGRAY;
+		pathInsideBitmap = false;
```

Comment:
why do you need pathInsideBitmap to determine wether the path is inside or outside?